SPECS-EXTENDED/opencsd/opencsd.spec

```diff
@@ -0,0 +1,106 @@
+Summary:        An open source CoreSight(tm) Trace Decode library
+Name:           opencsd
+Version:        1.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/Linaro/OpenCSD
+Source:         https://github.com/Linaro/OpenCSD/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  git
+BuildRequires:  make
+BuildRequires:  patch
+
+%description
+This library provides an API suitable for the decode of ARM(r)
+CoreSight(tm) trace streams.
+
+%package devel
+Summary:        Development files for the CoreSight(tm) Trace Decode library
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+The opencsd-devel package contains headers and libraries needed
+to develop CoreSight(tm) trace decoders.
+
+%prep
+%setup -q -n OpenCSD-%{version}
+
+%build
+cd decoder/build/linux
+export CFLAGS="%{optflags}"
+export CXXFLAGS="%{optflags}"
```

Comment:
These two lines are redundant in Mariner- [we actually inject these environment variables](https://github.com/microsoft/CBL-Mariner/blob/78e1d24d476592be6b4800b57017ad0cf6640925/SPECS/mariner-rpm-macros/macros#L48-L53) into the spec template as part of the `%build` macro.

```suggestion
```